lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -1144,4 +1146,44 @@ module.exports = function(self, options) {
     }
   };
 
+  self.addFixOrientationsMigration = function() {
+    self.apos.migrations.add(self.__meta.name + '.fixOrientations', function(callback) {
```

Comment:
Do not write new migrations in callback style. Migrations may return a promise.

Reply:
callback removed